### PR TITLE
Add Galley command-line flags "server" and "purge"

### PR DESCRIPTION
**Pull request by ozevren**
Add command-line flags for server and purge commands.

### CODE REVIEW

File: galley/pkg/server/mapping.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+package server
+
+import (
+	"k8s.io/apimachinery/pkg/runtime/schema"
+
+	"istio.io/istio/galley/pkg/crd"
+)
+
+var mappingData = map[schema.GroupVersion]schema.GroupVersion{
```
[the remark this answers is not in the document]
Comment (ozevren) [reply galley/pkg/server/mapping.go]: I don't think having this configurable (thus exposed externally) is useful. I'd rather keep this as an implementation detail.

**Comment by ozevren** [comment]
PTAL

File: galley/cmd/gals/cmd/purge.go [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+			if err := purgeResources(fatalf); err != nil {
+				fatalf("%v", err)
+			}
+		}}
+}
+
+func purgeResources(fatalf shared.FormatFn) error {
+	config, err := clientcmd.BuildConfigFromFlags("", flags.kubeConfig)
+	if err != nil {
+		fatalf("Error getting Kube config: %v", err)
```
Comment (ayj) [inline galley/cmd/gals/cmd/purge.go]: Return the error here and let the caller `purgeCmd` invoke `fatalf`?

Comment (ayj) [reply galley/cmd/gals/cmd/purge.go]: Similar comment applies to galley/cmd/gals/cmd/server.go:43

**Pull request merged**